tools/toktx/toktx.cc

```diff
@@ -724,7 +724,16 @@ toktxApp::main(int argc, _TCHAR *argv[])
             image =
               Image::CreateFromFile(infile,
                                     options.assign_oetf == KHR_DF_TRANSFER_UNSPECIFIED,
-                                    options.bcmp || options.bopts.uastc);
+                                    options.etc1s || options.bopts.uastc ||
+                                    (options.astc &&
+                                     options.astcopts.mode != KTX_PACK_ASTC_ENCODER_MODE_HDR));
+
+            if (options.astc &&
+                options.astcopts.mode != KTX_PACK_ASTC_ENCODER_MODE_HDR &&
+                image->getComponentSize() == 2) {
+                cerr << name << ": Warning! input file is 16bit but no HDR option provided."
```

Comment:
As I noted in reply to my original comment,, toktx should default to HDR for 16-bit inputs so this warning is not necessary.

Reply:
Fixed in https://github.com/KhronosGroup/KTX-Software/pull/433/commits/18af4983456764eb784babe2d11a125f5d2b7f5c